scripts/import-npa/natural_protected_areas.sh

```
@@ -0,0 +1,493 @@
+download_dataset() {
+
+  if [[ -z "$DATASET_URL" ]]; then
+    print_message error "Dataset URL (--url) or DATASET_URL environment variable is required"
+    exit 1
+  fi
+
+  print_message info "Preparing to download dataset from $DATASET_URL..."
+  
+  local zip_name="protected_planet_$(date +%Y%m%d%H%M%S).zip"
+  local zip_path="${TEMP_DIR}/${zip_name}"
+  local existing_zip=$(find "$TEMP_DIR" -name "protected_planet_*.zip" -type f | sort | tail -n 1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)